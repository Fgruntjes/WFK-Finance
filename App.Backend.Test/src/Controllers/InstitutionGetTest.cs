namespace App.Backend.Test.Controllers;

public class InstitutionGetTest : IClassFixture<InstitutionGetFixture>
{
	private readonly InstitutionGetFixture _fixture;

	public InstitutionGetTest(InstitutionGetFixture fixture)
	{
		_fixture = fixture;
	}

	[Fact]
	public async Task Success()
	{
		var result = await _fixture.ExecuteQuery(new { Id = _fixture.InstitutionId });
		result.MatchSnapshot();
	}

	[Fact]
	public async Task NotFound()
	{
		var result = await _fixture.ExecuteQuery(new { Id = Guid.NewGuid() });
		result.MatchSnapshot();
	}
}