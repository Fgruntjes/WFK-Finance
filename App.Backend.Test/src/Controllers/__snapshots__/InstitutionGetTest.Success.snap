{
  "data": {
    "institution": {
      "get": {
        "name": "MyFakeName-NL"
      }
    }
  }
}
